clients/privacy-center/features/common/settings.slice.ts

```diff
@@ -0,0 +1,41 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+
+import { useAppSelector } from "~/app/hooks";
+import { PrivacyCenterClientSettings } from "~/app/server-environment";
+import type { RootState } from "~/app/store";
+
+interface SettingsState {
+  settings?: PrivacyCenterClientSettings;
```

Comment:
nice, I like this being separated out 👍

Reply:
ah yeah, we do that elsewhere too. normally that's mitigated a bit since we query endpoint caches instead of saving in state i.e. https://github.com/ethyca/fides/blob/916952ef352f82f7ac904fdbe5e8be0747ed3243/clients/admin-ui/src/features/dataset/dataset.slice.ts#L157-L161

but I'm not sure if we can/would want to do that for server-environment